coupling/delwaq/util.py

```diff
@@ -0,0 +1,169 @@
+"""Utilities to write Delwaq (binary) input files."""
+
+import os
+import platform
+import struct
+import subprocess
+from datetime import timedelta
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from ribasim.utils import MissingOptionalModule
+
+try:
+    import xugrid
+except ImportError:
+    xugrid = MissingOptionalModule("xugrid")
+
+
+def strfdelta(tdelta):
```

Comment:
Most functions here have type hints on the arguments but not return type annotations. Would be a bit nicer for readability.